sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/ModelSettingsClient.cs

```diff
@@ -108,20 +105,20 @@ public virtual async Task<Response<TimeSeriesModelSettings>> UpdateModelSettings
         }
 
         /// <summary>
-        /// Updates model default type Id on Time Series model settings asynchronously.
+        /// Updates model name on Time Series model settings synchronously.
         /// </summary>
-        /// <param name="defaultTypeId">Default type Id of the model that new instances will automatically belong to.</param>
+        /// <param name="name">Model display name which is mutable by the user. Initial value is &quot;DefaultModel&quot;.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The updated model settings with the http response <see cref="Response{TimeSeriesModelSettings}"/>.</returns>
-        public virtual async Task<Response<TimeSeriesModelSettings>> UpdateModelSettingsDefaultTypeIdAsync(string defaultTypeId, CancellationToken cancellationToken = default)
+        public virtual Response<TimeSeriesModelSettings> UpdateName(string name, CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TimeSeriesInsightsClient)}.{nameof(UpdateModelSettingsDefaultTypeId)}");
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TimeSeriesInsightsClient)}.{nameof(UpdateName)}");
             scope.Start();
             try
             {
                 // To do: Generate client session Id
-                var options = new UpdateModelSettingsRequest { DefaultTypeId = defaultTypeId };
-                Response<ModelSettingsResponse> modelSettings = await _modelSettingsRestClient.UpdateAsync(options, null, cancellationToken).ConfigureAwait(false);
+                var options = new UpdateModelSettingsRequest { Name = name };
+                Response<ModelSettingsResponse> modelSettings = _modelSettingsRestClient.Update(options, null, cancellationToken);
```

Comment:
same comment here and other places for client session id

Reply:
removing the client session id all together per our offline conversation